src/plugins/workspace/public/components/workspace_menu/workspace_menu.tsx

```diff
@@ -67,68 +86,68 @@ export const WorkspaceMenu = ({ coreStart }: Props) => {
     setPopover(false);
   };
 
-  const workspaceToItem = (workspace: WorkspaceObject) => {
+  const workspaceUseCaseToItem = (
+    workspace: WorkspaceObject,
+    index: number,
+    workspaceURL: string
+  ) => {
+    const useCase = workspace.features
+      ?.map(getUseCaseFromFeatureConfig)
+      .filter(isNotNull)
+      .map((uc) => {
+        return {
+          name: uc,
+          // TODO: this should be a link to the use case overview page.
```

Comment:
Are we tracking this somewhere?

Reply:
@yubonluo I think we can remove this comment as overview pages are tracked separately.

Reply:
Sure, I will remove it.